autoload/codefmt/clangformat.vim

```diff
@@ -19,6 +19,10 @@ let s:plugin = maktaba#plugin#Get('codefmt')
 function! s:ClangFormatHasAtLeastVersion(minimum_version) abort
   if !exists('s:clang_format_version')
     let l:executable = s:plugin.Flag('clang_format_executable')
+    if !executable(l:executable)
```

Comment:
Easy enough to fix with an override. I also had to fix the clangformat vroom tests because google-java-format takes precedence, I guess.